MANIFEST.in

```diff
@@ -1,7 +1,6 @@
 include README.rst
 include CONTRIBUTING.md
 include AUTHORS
-include .travis.yml
```

Comment:
Should we now `recursive-include` the `.github` dir instead?

Reply:
To be honest, I think it's not needed to have the CI config in the source distribution file. Let's leave it.